### ISSUE TITLE
Nominate @AugustinMauroy to the Website Team

### ISSUE THREAD

**Issue by ovflowd** [nodejs/nodejs.org#5504]
Well, this, in specific, is a rollercoaster. Augustin has been involved in contributing to the Website Redesign since early 2022. He's been exceedingly active on anything Website-related.

Which ponders why this nomination is only happening now? At the end of giving context, when nominating someone and why they should be nominated.

Augustin is engaged, proactive and has tons of energy and motivation; Having said that, it took a lot of work to communicate with them on their early contributions and create expectations. Fast-forwarding to this month, after a lot of feedback from numerous collaborators, TSC-members; Augustin has a better understanding of the collaboration model, and I find it easier to collaborate with them.

Where Augustin lacks in knowledge of Web Development, he exceeds in engagement. We all make mistakes and can learn with everyone in the room. And Augustin's engagement and curiosity often bring benefits.

I want to give this opportunity to Augustin as a learning opportunity and reward his immense dedication to the project. We are thankful.

For Augustin, please take this as an opportunity and act with humility; we all appreciate your contributions but keep in mind all the feedback and guidance we shared over time :)

[Pull requests opened throughout the Node.js organization](https://github.com/search?q=author%3AAugustinMauroy+org%3Anodejs&type=pullrequests)
[Comments on pull requests throughout the Node.js organization](https://github.com/search?q=commenter%3AAugustinMauroy+org%3Anodejs&type=pullrequests)
[Comments on issues throughout the Node.js organization](https://github.com/search?q=commenter%3AAugustinMauroy+org%3Anodejs&type=issues)
Comments on pull requests and issues in the [nodejs/nodejs.org](https://github.com/nodejs/nodejs.org/issues?q=commenter%3AAugustinMauroy+sort%3Aupdated-desc+) repository

cc @AugustinMauroy 
cc @nodejs/website

**Comment by bmuenzenmeyer** [comment]
I'm in full support. It's great to see @AugustinMauroy investing in himself and open source. I'll be here to help in any way I can.

**Comment by AugustinMauroy** [comment]
Thanks claudio, thanks Brian and all contributor.
I love nodeJS project and all members.
I'd be delighted to commit to joining the website team.

**Comment by shanpriyan** [comment]
+1 for @AugustinMauroy 🚀 , He's been actively involved in website contributions.

**Comment by ovflowd** [comment]
FYI: I'm waiting for @AugustinMauroy to be back from where he is right now (he's not at home) and let us know if he can contribute mostly because he mentioned that his computer stopped working, which sadly can strongly interfere with his ability to contribute for now.

Let's keep this on hold till then as we don't need to rush here, he's welcome at any time to join the team :)

**Comment by ovflowd** [comment]
Augustin has been unavailable for the time being, so I'm closing this issue for now (on hold), we'll reopen the nomination as soon as he's back :)

**Comment by ovflowd** [comment]
FYI I've talked with @AugustinMauroy and he opted to join the team only once we have the new design work ready to be worked on!

**Comment by AugustinMauroy** [comment]
Hey, I'd be delighted to join the team now and help move the project forward.

**Comment by ovflowd** [comment]
I've sent an invite to @AugustinMauroy now that we've updated the teams :)

**Comment by AugustinMauroy** [comment]
I'm delighted to be joining the team and moving the project forward with you all... 😁
